doc/whats_new.rst

```diff
@@ -34,6 +34,10 @@ API
 - ``Solver.skip`` can now be used to skip objectives that are incompatible
   for the Solver, by `Thomas Moreau`_ (:gh:`113`).
 
+- ``Solver`` can now use ``stop_strategy = 'callback'`` to allow for
```

Comment:
```suggestion
- ``Solver`` can now uses ``stop_strategy = 'callback'`` to allow for
```

Reply:
`use` is actually correct.

Reply:
@agramfort  got carried by the momentum of adding `s` to all my docstrings :rofl: